code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4197,3 +4195,89 @@
 /obj/item/weapon/reagent_containers/food/snacks/frog_leg/New()
 	..()
 	reagents.add_reagent(NUTRIMENT, 6)
+
+
+/obj/item/weapon/reagent_containers/food/snacks/pie/welcomepie
+	name = "friendship pie"
+	desc = "Offered as a gesture of Vox goodwill." //"Goodwill"
+	icon_state = "welcomepie"
+	bitesize = 4
+	New()
+		..()
+		reagents.add_reagent(SACID,6)
+		reagents.add_reagent(NUTRIMENT,2)
+
+/obj/item/weapon/reagent_containers/food/snacks/zhulongcaofan
+	name = "zhu l�ng cao f�n"
```

Comment:
Works everywhere except when moused over
![2016-06-29_01-01-07](https://cloud.githubusercontent.com/assets/6526157/16440093/f9ebe21e-3d94-11e6-851f-756a31d31f5b.png)

That'd be, everywhere except here
![dreamseeker_2016-06-29_01-01-39](https://cloud.githubusercontent.com/assets/6526157/16440097/0aeac440-3d95-11e6-8b94-542d748200d1.png)